dedibox-network/domains/how-to/enable-dnssec.mdx

```diff
@@ -28,16 +28,20 @@ DNSSEC should only be used by experienced users, due to the propagation time of
   - Not all registries support the same algorithms
   - Not all DNS servers (clients) verify DNSSEC, you can achieve responses on them despite bad DNSSEC configuration
 
+<Message type="important">
+ This page shows you how to enable DNSSEC for **Online domains only**. Find out how to enable DNSSEC for Domains and DNS domains via [the API](https://www.scaleway.com/en/developers/api/domains-and-dns/registrar-api/#path-domains-update-domain-dnssec) or the [Scaleway console](/network/domains-and-dns/how-to/configure-dnssec-internal-domain/).
```

Comment:
```suggestion
 This page shows you how to enable DNSSEC for **Online.net domains only**. Find out how to enable DNSSEC for Domains and DNS domains via [the API](https://www.scaleway.com/en/developers/api/domains-and-dns/registrar-api/#path-domains-update-domain-dnssec) or the [Scaleway console](/network/domains-and-dns/how-to/configure-dnssec-internal-domain/).
```

Reply:
@bene2k1 I moved your suggestion to this sentence, as I was not sure about putting the "Online.net" in the title.